.github/workflows/test.yml

```diff
@@ -58,7 +60,8 @@ jobs:
     - name: Run tests
       env:
         TEST_COVERAGE: '1'
-      run: /usr/bin/prove -I../test-nginx/lib -r t/
+      run: /usr/bin/prove -I../test-nginx/lib t/*.t
+      # TODO: run test with apisix-openresty
```

Comment:
Submit a new PR in the future to solve this problem?

Reply:
Yes